clients/xnat/README.md

```diff
@@ -0,0 +1,10 @@
+# XNAT API
+## Generate an OpenAPI client from the xnat spec
+1. Run the openapi generator tool:
+```shell
+docker run --rm -v $(pwd):/local openapitools/openapi-generator-cli generate -i /local/spec/xnat-api.yaml -g python -o /local/generated
+```
+2. update the dateutil dependency version in `generated/setup.py`:
+```text
+    "python-dateutil ~= 2.8.0",
+```
```

Comment:
I can fix the test (some minor changes to the api since I wrote that original test), but I'm wondering whether it's even worth keeping it there as I have written a separate unit test to test the helper method in `xnat_service.py` that calls the open api client (and essentially tests much the same thing as this test).

Reply:
Cool, removing it is fine :+1:

Reply:
removed it